### ISSUE TITLE
Site has many accessibility issues

### ISSUE THREAD

**Issue by GrantASL19** [ASL-19/civicdr#35]
The site has many accessibility issues, including:

- Focus outlines are hidden, which makes keyboard navigation impracticable. If the browser/OS has keyboard navigation enabled (on OS X, System Preferences -> Keyboard -> Shortcuts -> Full Keyboard Access), the user should be able to navigate the site using Tab, Shift-Tab, Space, Return, and the arrow keys.

- Actions that modify the view state don’t move the focus or notify the user that the state has changed. Navigating without visual context (e.g. with a screen reader like VoiceOver) is impracticable.  e.g.:

    - In the header, the profile icon link registers as “Link: Profile”. If non-sighted user clicks on this, they would be correct in assuming this would navigate to a profile page. Instead, nothing happens – they would have to manually navigate around to realize that clicking the link caused the “View Profile” and “Logout” links to appear. The site should focus the first menu item when the menu opens, and capture/retain the focus while it’s open.

    - If a user finds the “View Profile” link and clicks it, again, nothing happens from their perspective. The main view changes, but if they can’t see it, the only way for them to know would be to navigate around the page and notice that the content has changed. Their focus remains on the “View Profile” link, and they get no feedback. Instead, the site should focus a descriptive `<h1>` when the view loads.

    - If a user clicks the “Edit” button on the profile page, nothing happens. To access the overlay, they would have to navigate back/up, since the overlay appears before the “Edit” button in the DOM, and even if they reached the overlay, it wouldn’t necessarily be clear to them what it is, or why they’re suddenly backing into “Cancel” and “Save” buttons.

- The alt text for the main logo (inside the header `<h1>`) is “logotype“. It should be “Civil Society Centre for Digital Resilience”. The same applies to other images on the site, if applicable.

- The structure and flow of the login page is awkward without visual context, especially if 2FA is enabled and the user isn’t fully familiar with it. I figure the Auth0 stuff might be outside of your control though.

This is only what I’ve found on the first few pages on the service provider UI. Accessibility may be outside of the development scope at the moment, but I figured I should document it so it’s on the radar in the future.

**Comment by tiffrobo** [comment]
Closing this ticket and we will make a UI enhancement ticket for 'link'
